### DISCUSSION TITLE
Relaxation of jump value

### DISCUSSION THREAD

**Discussion by styyokuda** [idaholab/moose#26698] [Q&A General]
All,
The value of my model jumps sharply at the interface and it requires a very small time step and  a very small mesh size to get a converged solution.  I am using a PenaltyInterfaceDiffusion type of an object at the interface.  I would like to ask if MOOSE has some apps to relax this jump: for example, gradually increase the jump value as a function of time? Please help. S. Thomas

**Comment by GiudGiud** [comment]
Hello

I would make the parameter for the value of the jump controllable. Then you can use the Controls system to slowly increase the value through time.
To make a parameter controllable, you just have to make the parameter a Real and have
```
params.declareControllable("param_name");
```
in the validParams() of the object

Guillaume

**Comment by Rahim-Habibi** [comment]
Hi @styyokuda,
is the jump like what i explained here #26670 ?
I dont know your model enough, but in my case it is a 2D interface too. 
Rahim
